projects/components/src/titled-content/titled-content.component.ts

```diff
@@ -75,3 +86,13 @@ export const enum TitlePosition {
   Header = 'header',
   Footer = 'footer'
 }
+
+export const enum TitledContentTitleStyle {
+  Regular = 'regular',
+  GrayedOut = 'grayed-out'
```

Comment:
Can you please add a comment that would indicate which one refers to the new design?

Reply:
added